src/Microsoft.Win32.Registry/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,21 @@
+advapi32.dll!RegEnumValueW
```

Comment:
Isn't that hammer too big? We need to have a property to just disable the PInvokeChecker and not analyzers.